internal/dag/gatewayapi_processor.go

```diff
@@ -225,14 +252,15 @@ func (p *GatewayAPIProcessor) Run(dag *DAG, source *KubernetesCache) {
 
 		// Process all the HTTPRoutes that match this Gateway.
 		for _, matchingRoute := range matchingHTTPRoutes {
-			p.computeHTTPRoute(matchingRoute, listenerSecret, listener.Hostname)
+			p.computeHTTPRoute(matchingRoute, listenerSecret, listener.Hostname, errs)
```

Comment:
I see you added `errs` as an arg to `computeHTTPRoute` and `computeTLSRoute` but those aren't used. Should we change this to be just a boolean to simply like `validGateway` or something? 

If not would be nice to describe what `errs` are a bit maybe in the name to make it clear what they mean and where they come from down in the methods.

Reply:
I'll update the PR to pass a bool that indicated gateway validity to the compute methods.